ee/fleetd-chrome/src/tables/screen_lock.ts

```diff
@@ -0,0 +1,20 @@
+import Table from "./Table";
+export default class TableScreenLock extends Table {
+  name = "screen_lock";
+  columns = ["delay"];
+
+  async generate() {
+    let delay;
+
+    try {
+      const autoLockDelay = await new Promise((resolve) =>
+        chrome.idle.getAutoLockDelay(resolve)
+      );
+      delay = autoLockDelay;
+    } catch (err) {
+      console.warn("get screen lock info:", err);
+    }
+
+    return [{ delay }];
```

Comment:
If `chrome.idle.getAutoLockDelay` fails, instead of returning an `undefined`, we should return the error to Fleet (so it's displayed to the user running the live query). Maybe not capture the error and see if it makes it so that Fleet receives the error?